File: code/fizz buzz by cpp/01.cpp
#include <stdio.h>

int main(int argc, char* argv[]) {

	for (int i = 0; i < 20; i++) {

		printf("%02u: ", i);

		if (i % 3 == 0)
			printf("fizz");
		if (i % 5 == 0)
			printf("buzz");

		printf("\n");
	}
	return 0;
}

File: code/fizz buzz by Rust/01.rs
fn main() {

	for n in 1..19 {

		print!("{:02}: ", n);

		if n % 3 == 0 {
			print!("fizz");
		}

		if n % 5 == 0 {
			print!("buzz");
		}

		println!("");
	}
}
